dapps/multisig-toolkit/src/components/preview-effects/utils.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+export const onChainAmountToFloat = (amount: string, decimals: number) => {
+	if (!decimals) return undefined;
```

Comment:
Is it expected that `0` returns undefined? Also generally prefer null over undefined for explicit empty values